guides/source/testing.md

```diff
@@ -671,6 +671,16 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
 end
 ```
 
+If you want to use a headless browser, you could use Headless Chrome by adding headless_chrome in the `:using` argument.
```

Comment:
Do you think "headless_chrome" should be in ``?

Reply:
👍